lib/aquatone/collectors/wayback.rb

```diff
@@ -0,0 +1,23 @@
+require 'uri'
+
+module Aquatone
+  module Collectors
+    class Crtsh < Aquatone::Collector
```

Comment:
The class name should probably be something like `WaybackMachine`

Reply:
Oh, whoops!